src/mobile/src/main/java/tupperdate/android/appbars/mainBottomBar.kt

```diff
@@ -4,31 +4,45 @@ import androidx.compose.foundation.Icon
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.material.BottomAppBar
+import androidx.compose.material.Button
 import androidx.compose.material.IconButton
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
 import tupperdate.android.R
 
 @Composable
-fun mainBottomBar() {
+fun mainBottomBar(
+    onLike: () -> Unit,
+    onDislike: () -> Unit,
+    onReturn: () -> Unit,
+    onRecipeClick: () -> Unit
+) {
     BottomAppBar(backgroundColor = Color.White)
     {
         Row(Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceEvenly)
         {
-            IconButton(onClick = {}) {
-                Icon(vectorResource(id = R.drawable.ic_navigate_before_black_18dp))
+            Button(modifier = Modifier.size(45.dp), onClick = onReturn, shape = CircleShape, backgroundColor = Color(
+                197, 180, 246)) {
```

Comment:
Extract to resources if not already in our theme

Reply:
it's done